### ISSUE TITLE
[Bug] Item module won't load causing LUA errors

### ISSUE THREAD

**Issue by NoNick96** [fang2hou/ElvUI_WindTools#254]
**Describe the bug**
After loading in or reloading the UI I get a bunch of LUA errors and the Item module from WindTools isn't loading in. The rest of the modules are working fine.

**To Reproduce**
Reload the UI

**Expected behavior**
The module should load in

**Screenshots**
![Wow_Tpjd0LGgIC](https://user-images.githubusercontent.com/16721863/201456808-4ed366be-3062-4a70-be33-fecb967b8394.png)

**Information (you can find these in [WindTools-Information]):**
 - Addon Version: 3.00
 - ElvUI Version 13.01
 - WoW Build 10.0.0 (46597)

**Additional context**
LUA errors:
```
6x ...AddOns/ElvUI_WindTools/Modules/Item/AlreadyKnown.lua:2: attempt to index field 'ItemSearch' (a nil value)
[string "@Interface/AddOns/ElvUI_WindTools/Modules/Item/AlreadyKnown.lua"]:2: in main chunk

Locals:
W = <table> {
 SetDefaultModuleLibraries = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:363
 EnableModule = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:147
 IterateEmbeds = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:438
 SecureHookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:409
 ConstructCompatibilityFrame = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Compatibility.lua:17
 RawHookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:394
 Printf = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceConsole-3.0/AceConsole-3.0.lua:65
 AddCommand = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Commands.lua:22
 SetDefaultModulePrototype = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:405
 defaultModuleState = true
 IsEnabled = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:447
 ScheduleTimer = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:94
 DisableModule = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:346
 RegisterModule = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Core.lua:93
 RegisteredModules = <table> {
 }
 RegisterMessage = <function> defined @Interface/AddOns/BigWigs/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:90
 UnregisterMessage = <function> defined @Interface/AddOns/BigWigs/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:153
 hooks = <table> {
 }
 CompatibleFont = false
 SetEnabledState = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:273
 enabledState = true
 RegisterEvent = <function> defined @Interface/AddOns/BigWigs/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:90
 Utilities = <table> {
 }
 IterateModules = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:433
 Version = "3.00"
 RawHook = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:312
 Media = <table> {
 }
 Enable = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:290
 UnregisterAllEvents = <function> defined @Interface/AddOns/BigWigs/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:174
 PlainTitle = "WindTools"
 Locale = "enUS"
 GetModule = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:206
 defaultModuleLibraries = <table> {
 }
 TimeLeft = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:172
 UnregisterChatCommand = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceConsole-3.0/AceConsole-3.0.lua:107
 ScheduleRepeatingTimer = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:129
 CancelAllTimers = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:160
 SupportElvUIVersion = 13
 GetName = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:275
 ChineseLocale = false
 name = "ElvUI_WindTools"
 Initialize = <function> defined @Interface/AddOns/ElvUI_WindTools/Initialize.lua:44
 Fang2houUIPrivate = <function> defined @Interface/AddOns/ElvUI_WindTools/Settings/Fang2houUI/Private.lua:5
 orderedModules = <table> {
 }
 Disable = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:310
 CheckInstalledVersion = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Core.lua:136
 UpdateScripts = <


6x Interface/AddOns/ElvUI_WindTools/Options/Item.lua:6: Usage: GetModule(name, silent): 'name' - Cannot find module 'AlreadyKnown'.
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:208: in function `GetModule'
[string "@Interface/AddOns/ElvUI_WindTools/Options/Item.lua"]:6: in main chunk

Locals:
(*temporary) = "Usage: GetModule(name, silent): 'name' - Cannot find module 'AlreadyKnown'."

6x Interface/AddOns/ElvUI_WindTools/Modules/Item/AlreadyKnown.lua:1 (null)

6x Interface/AddOns/ElvUI_WindTools/Options/Item.lua:1 (null)

6x FrameXML\Bindings.xml:1 Interface/AddOns/ElvUI_WindTools/Modules/Item/AlreadyKnown.lua:1 (null)

6x FrameXML\Bindings.xml:1 Interface/AddOns/ElvUI_WindTools/Options/Item.lua:1 (null)
```

**Comment by fang2hou** [comment]
elvui make a very big update in 13.01, just wait for the fix version of ElvUI_WindTools.

**Comment by NoNick96** [comment]
#252

**Comment by fang2hou** [comment]
fixed in [`3.01`](https://github.com/fang2hou/ElvUI_WindTools/releases/tag/3.01)
